### DISCUSSION TITLE
Issues with BS5 tab nav

### DISCUSSION THREAD

**Discussion by lilpxl** [understrap/understrap-child#348] [Q&A] [answered]
Hi everyone!

Just wanted to ask here before I go down the rabbit hole. I set up a tab nav using the exact Bootstrap 5 demo code shown [here](https://getbootstrap.com/docs/4.1/components/navs/#javascript-behavior) but the nav links don't do anything. I see it says you need to load `util.js` but I am assuming thats already meant to be loading in the Understrap Child theme. Anyways, I haven't dug in too far and wanted to see if I was missing something before I burned a bunch of time trying to sort it. really appreciate any help - the exact code I am pasting in is pasted below. Thank you!

```
<ul class="nav nav-pills mb-3" id="pills-tab" role="tablist">
  <li class="nav-item">
    <a class="nav-link active" id="pills-home-tab" data-toggle="pill" href="#pills-home" role="tab" aria-controls="pills-home" aria-selected="true">Tab One</a>
  </li>
  <li class="nav-item">
    <a class="nav-link" id="pills-profile-tab" data-toggle="pill" href="#pills-profile" role="tab" aria-controls="pills-profile" aria-selected="false">Tab Two</a>
  </li>
  <li class="nav-item">
    <a class="nav-link" id="pills-contact-tab" data-toggle="pill" href="#pills-contact" role="tab" aria-controls="pills-contact" aria-selected="false">Tab Three</a>
  </li>
</ul>
<div class="tab-content" id="pills-tabContent">
  <div class="tab-pane fade show active" id="pills-home" role="tabpanel" aria-labelledby="pills-home-tab">Content One</div>
  <div class="tab-pane fade" id="pills-profile" role="tabpanel" aria-labelledby="pills-profile-tab">Content Two</div>
  <div class="tab-pane fade" id="pills-contact" role="tabpanel" aria-labelledby="pills-contact-tab">Content Three</div>
</div>
```

**Comment by bacoords** [comment]
Are you using BS4 or 5? You mention 5 but I see attributes like `data-toggle`. In BS5 those have become `data-bs-toggle`

**Comment by lilpxl** [comment, marked as answer]
I wonder if their documentation is out of date as [the code above is from their page](https://getbootstrap.com/docs/4.1/components/navs/#javascript-behavior) and uses `data-toggle`. I actually went down a different path for my project but still curious if something is amiss. Thanks for your reply!

EDIT: That code is indeed from Bootstrap 4 - pretty sure they linked me to that page from the BS5 docs but either way this is my bad. Thanks for your help tho!

**Reply by bacoords** [reply]
Sounds good! I was just coming here to confirm that you were checking the BS version on the docs page but it looks like you got it figured out.
